src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/C14NAncestralNamespaceContextManager.cs

```diff
@@ -84,14 +84,14 @@ internal override void GetNamespacesToRender(XmlElement element, SortedList attr
             }
         }
 
-        internal override void TrackNamespaceNode(XmlAttribute attr, SortedList nsListToRender, Hashtable nsLocallyDeclared)
+        internal override void TrackNamespaceNode(XmlAttribute? attr, SortedList nsListToRender, Hashtable nsLocallyDeclared)
         {
-            nsLocallyDeclared.Add(Utils.GetNamespacePrefix(attr), attr);
+            nsLocallyDeclared.Add(Utils.GetNamespacePrefix(attr!), attr);
```

Comment:
Why are you sure that `attr` is not `null`, is it's explicitly marked as nullable?

Reply:
Good point. `TrackNamespaceNode(XmlAttribute? attr...` doesn't the nullable.